src/Our.Umbraco.Vorto/Web/UI/App_Plugins/Vorto/js/vorto.js

```diff
@@ -192,6 +202,10 @@
             }
         });
 
+        $scope.$watch("model.value.values", function () {
```

Comment:
"ActiveLanguage" changes whenever you switch language, so you would see the marks when you move between languages. Doing it this way would just limit the number of checks to only happen once every time you switch language.